intellij-plugin/src/main/resources/inspectionDescriptions/ApolloOneOfInputCreation.html

```diff
@@ -0,0 +1,8 @@
+<html>
+<body>
+Reports invalid constructor invocations of <code>@oneOf</code> input types.
+<p>
+    Exactly one field must be set, and it must be <code>Present</code>.
```

Comment:
```suggestion
    Exactly one input field must be set, and it must be <code>Present</code>.
```

Reply:
Or maybe more like so?

```suggestion
    Constructor of `@oneOf` input class must have exactly one <code>Present</code> argument.
```

* Field is JVM/GraphQL. 
* Property is Kotlin
* Constructor takes arguments?

Reply:
Yup, much better!